rs/execution_environment/src/canister_manager/types.rs

```diff
@@ -243,7 +243,11 @@ pub struct InstallCodeContext {
     pub canister_id: CanisterId,
     pub wasm_source: WasmSource,
     pub arg: Vec<u8>,
+    // EXC-370: The following fields will be removed once `compute_allocation`
+    // `memory_allocation` are removed from `InstallCodeArgs`.
+    #[allow(dead_code)]
```

Comment:
Attempting to remove those now, would result in the need to update lots of tests in `canister_manager` which I wanted to avoid and include only in the follow-up change.